src/helpers/createFeatureAPI.js

```diff
@@ -129,6 +130,27 @@ export default (enable, store) => features => {
         store.dispatch(actions.setThumbnailMultiselect(enable));
       },
     },
+    [Feature.PageNavigation]: {
+      fn: () => {
+        const { up: arrowUpHandler, down: arrowDownHandler } = hotkeysManager.keyHandlerMap;
```

Comment:
i think we need to add in page up and page down as well? as i can still navigate the page?